charts/actions-runner-controller/values.yaml

```
@@ -270,3 +270,5 @@ githubWebhookServer:
     enabled: false
     # minAvailable: 1
     # maxUnavailable: 3
+  ## Queue limit size override, by default its 100
+  # queueLimit: 1000
```

Above `charts/actions-runner-controller/README.md` says default value set in `values.yaml` is `100` but we are not setting any value.

Lets either set the default value to 100 here or update `README.md` above to blank.

---

Should we refactor to move this default value out of `horizontal_runner_autoscaler_webhook.go` to `values.yaml`? Moving this default config to `values.yaml` will give users more clarity. WDYT?

---

Good point!

What I had in my mind was that one can still deploy ARC by writing their own K8s manifest yaml or their own Helm chart . The hard-coded default value can be handy and user-friendly in that case.
Perhaps we can just make it in sync with the coded default like:

```suggestion
  # queueLimit: 100
```

and add some code comment in horizontal_runner_autoscaler_webhook.go to let the programmer know that they need to change the default value included in the default values.yaml along with the hard-coded default?

WDYT?

---

> one can still deploy ARC by writing their own K8s manifest yaml or their own Helm chart

Got it, ya in that case lets keep that default in our code and update the README.md